src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -387,7 +410,8 @@ export function MapAnalysisImpl(props: Props & CompleteAppState) {
                 onAnalysisNameEdit={analysisState.setName}
                 onToggleExpand={() => setMapHeaderIsExpanded((c) => !c)}
                 studyName={studyRecord.displayName}
-                totalEntitesCount={totalEntityCount}
+                totalEntitesInSampleCount={totalEntitiesInSampleCount}
+                totalEntitiesInSubsetCount={totalEntityInSubsetCount}
                 visibleEntitiesCount={totalVisibleEntityCount}
```

Comment:
We will also need to pass at least the name of the outputEntity to the header component too.

We should probably (in a new ticket) make this a drop-down select menu.

But for now, you can get the name of the output entity as follows:

Ah... I see, you pass the StudyEntity object itself (e.g. `outputEntity` returned from `useMapMarkers`) and we have `makeEntityDisplayName` function in study-metadata.ts which will format the plural for you.

Reply:
This commit replaces the hardcoded "samples" with the plural display name value of `outputEntity`: https://github.com/VEuPathDB/web-eda/pull/1656/commits/3a8aabcfc36c4cec9ff99539be7fa4dfca53a0d0